### PR TITLE
Improved launch screen.

### PR DESCRIPTION
**Pull request by Metaxa007**

### CODE REVIEW

**Comment by TomTasche** [comment]
TODO for me: provide higher resolution app icon

File: OpenDocumentReader/Assets.xcassets/launch_image.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    },
+    {
+      "idiom" : "iphone",
+      "filename" : "launch_image.png",
```
[the remark this answers is not in the document]
Comment (Metaxa007) [reply OpenDocumentReader/Assets.xcassets/launch_image.imageset/Contents.json]: i guess i can, but i am not sure, that is a best practise to use the same image for all devices.

**Comment by Metaxa007** [comment]
Was not able to reproduce the problem on iPhone 8.

**Comment by Metaxa007** [comment]
ezgif com-video-to-gif
Was not able to reproduce the problem on the iPhone 8.

**Comment by Metaxa007** [comment]
"no icon on iPad air."
I did no set the image, because did not have one that was big enough(resolution). Can reuse the same, that we use for iPhones.

**Comment by TomTasche** [comment]
Regarding iPhone 8: nice, that only seems to happen if you tilt the device to the left (you tilted to the right). Moreover, the launch screen is upside down if you tilt the phone upside down (not important to fix right now, just an observation).

Regarding icons: please use the one I linked to earlier. Doesn't matter if you duplicate it or reference the original. That's the highest resolution we have right now. Is that enough? @Metaxa007

**Comment by Metaxa007** [comment]
@TomTasche 
tried to tilt also to the left, but it works fine on iPhone 8. Do you use a physical device or simulator? 
Also added a universal image set.

**Comment by TomTasche** [comment]
Weird, now it doesn't work the other way around. Minor bug, we can ignore that for now!

**Pull request merged**